Geometry/HGCalCommonData/plugins/DDHGCalHEAlgo.cc

```diff
@@ -242,11 +246,13 @@ void DDHGCalHEAlgo::constructLayers(const DDLogicalPart& module,
 		     DDSplit(materials_[ii]).second);
       DDMaterial matter(matName);
       DDLogicalPart glog;
-      if (layerSense_[ly] == 0) {
+      if (layerSense_[ly] < 1) {
 	double alpha = CLHEP::pi/sectors_;
+	int    nsec  = (layerSense_[ly] == 0 || absorbMode_ == 0) ? 2 : 2;
```

Comment:
Same question about `nsec`

Reply:
nsec need to be a variable. All the conditions for a change of nsec was not summarized while the PR is made. It is needed to integrate this PR quickly and appropriate logic will be provided in the next PR

Reply:
In the future, to facilitate more rapid PR approval, please comment any tricky, provisional, or potentially confusing lines of code. A comment can tell a reviewer that an odd-looking line of code is intended and not a mistake.

Reply:
Thanks - I shall